tests/Agent/UnitTests/Core.UnitTest/Configuration/DefaultConfigurationTests.cs

```diff
@@ -1871,8 +1874,9 @@ public void ApplicationNamesPullsSingleNameFromIisExpressSitenameEnvironmentVari
 
             NrAssert.Multiple(
```

Comment:
Not related to your code: I'm not sure I like this assert helper... It makes it difficult to see the exact line that fails, and is more code than just a list of asserts 🤷‍♂️